controls/mousestates/cMouseNormalState.cpp

```diff
@@ -0,0 +1,195 @@
+#include <algorithm>
+#include "d2tmh.h"
```

Comment:
It's better to include only what is used in this file. Start with `#include "mousestates/cMouseNormalState.h"` and then add what's missing.

Reply:
this was laziness from my part. I'll try to get it working without `d2tmh.h`

Reply:
Well, I try to get through this mess, since I need to include `cPlayer.h` somewhere, it brings in the whole world and pain. I'll try to spend a bit more time on it, but if it is too hard, I will delegate that to a separate PR/ticket.

Reply:
Sure. This PR is already pretty big. If it results in changing multiple files, we can just add this tech debt.